### ISSUE TITLE
MultKAN.reg() got an unexpected keyword argument 'entropy_offset'

### ISSUE THREAD

**Issue by Akaqox** [KindXiaoming/pykan#489]
On Physics_2B_conservation_law_2D.ipynb tutorial notebook on the section:
`   reg_loss = model.reg(lamb_l1=1., entropy_offset=1e-4, lamb_coef=1.)`

it says MultKAN.reg() got an unexpected keyword argument 'entropy_offset' and if I am remove this statement and retry as

it says
MultKAN.reg() missing 3 required positional arguments: 'reg_metric', 'lamb_entropy', and 'lamb_coefdiff'

I dont know how to handle these arguments and what is the best values for this tutorial problem.

**Comment by Akaqox** [comment]
I understand now it means regularization and we can choose between

 reg_metric : the regularization metric
                'edge_forward_spline_n', 'edge_forward_spline_u', 'edge_forward_sum', 'edge_backward', 'node_backward'
            lamb_l1 : float
                l1 penalty strength
            lamb_entropy : float
                entropy penalty strength
            lamb_coef : float
                coefficient penalty strength
            lamb_coefdiff : float
                coefficient smoothness strength
 I hope that when you are free, you can fix the tutorial.
